### DISCUSSION TITLE
Curiosity in godot_rl_agents?

### DISCUSSION THREAD

**Discussion by gogela** [edbeeching/godot_rl_agents#173] [Ideas]
Is there any plan to support curiosity (RND, ICM,...) directly in godot RL agents (perhaps as another SB3 wrapper)?
Or is there any other recommended way of implementing it?

**Comment by edbeeching** [comment]
If you can find an open-source implementation in python it should be fairly easy to integrate. I found an example repo here: https://github.com/RLE-Foundation/rllte
